shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/main/java/org/apache/shardingsphere/mode/manager/cluster/ClusterContextManagerBuilder.java

```diff
@@ -83,8 +84,14 @@ private Properties getTransactionProperties(final MetaDataContexts metaDataConte
                 ? TransactionConfigurationFileGeneratorFactory.newInstance(transactionRule.get().getProviderType()) : Optional.empty();
         if (schemaName.isPresent() && fileGenerator.isPresent()) {
```

Comment:
please revert the condition judge, and return `new Properties()` firstly.

Reply:
I don't change the condition judge, just fill the props to `MetaDataContexts`.